### ISSUE TITLE
error: loop variable 'hitSeeds' of type 'const std::__1::vector<unsigned int, std::__1::allocator<unsigned int> >' creates a copy from type 'const std::__1::vector<unsigned int, std::__1::allocator<unsigned int> >'

### ISSUE THREAD

**Issue by yurivict** [bcgsc/abyss#364]
Version 2.3.0 fails to compile with clang-10:
```


RAlgorithmsShort.cpp:275:20: error: loop variable 'hitSeeds' of type 'const std::__1::vector<unsigned int, std::__1::allocator<unsigned int> >' creates a copy from type 'const std::__1::vector<unsigned int, std::__1::allocator<unsigned int> >' [-Werror,-Wrange-loop-construct]
                        for (const auto hitSeeds : g_spacedSeedsBloom->contains(sequence)) {
                                        ^
RAlgorithmsShort.cpp:275:9: note: use reference type 'const std::__1::vector<unsigned int, std::__1::allocator<unsigned int> > &' to prevent copying
                        for (const auto hitSeeds : g_spacedSeedsBloom->contains(sequence)) {
                             ^~~~~~~~~~~~~~~~~~~~~ 
                                        &
```

OS: FreeBSD 12.2

**Comment by schutzekatze** [comment]
Thanks for reporting this! I've fixed the error in master branch and have added clang-10 to the CI so we catch errors like this in the future.

Cheers

**Comment by schutzekatze** [comment]
Fixed in version 2.3.1.
